docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -409,27 +409,18 @@ Post-Release Review Process
 
      XXX: This step should be automated. See `FLOC-1039 <https://clusterhq.atlassian.net/browse/FLOC-1039>`_.
 
-   * The node package (``flocker-node``) should be installed on all supported platforms.
-     You can use vagrant to boot a clean Fedora 20 machine as follows:
+   * The node package (``flocker-node``) should be installed and tested on all supported platforms.
 
-     .. code-block:: console
-
-        mkdir /tmp/fedora20
-        cd /tmp/fedora20
-        vagrant init clusterhq/fedora20-updated
-        vagrant up
-        vagrant ssh
+     The acceptance tests can be run automatically using the ``run-acceptance-tests`` tool.
+     See :ref:acceptance-testing for details of the configuration files required for each platform.
```

Comment:
:question: Perhaps instead of linking to the document, which starts with an unnecessary-for-this explanation of how the acceptance test runner works, each needed section could be linked to.